dbt_copilot_helper/commands/conduit.py

```diff
@@ -316,14 +354,16 @@ def start_conduit(application: Application, env: str, addon_type: str, addon_nam
 @click.argument("addon_name", type=str, required=True)
 @click.option("--app", help="AWS application name", required=True)
 @click.option("--env", help="AWS environment name", required=True)
-def conduit(addon_name: str, app: str, env: str):
+@click.option("--write", is_flag=True, help="Allow write access to database addons")
+@click.option("--admin", is_flag=True, help="Allow admin access to database addons")
```

Comment:
At the moment `--write` overrides `--admin` if both are passed.

I feel like passing both is a mistake and it might be better to exit with an error.

Reply:
Hmm admin should be taking precedence if both are provided but I agree having both doesn't feel right

Reply:
How about if this was just "--access <read|write|admin>" rather than boolean flags?

Reply:
Yeah I'll have a look at that, maybe with read as the default so you don't always have to specific the access